File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/PackageSpecifications/IPackageSpecificationData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.PackageSpecifications
{
    public interface IPackageSpecificationData :  CoreObject.IObjectID, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IPackageSpecificationSpecification, IPackageSpecificationValue
    {

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/PackageSpecifications/IPackageSpecificationData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\PackageSpecifications;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IPackageSpecificationData extends IPackageSpecificationSpecification,IPackageSpecificationValue,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
